pkgs/by-name/mi/microsocks/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+}:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
{ stdenv,
  fetchFromGitHub,
  lib,
}:

stdenv.mkDerivation rec {
```

Reply:
Ah.. my alejandra formatter doesn't  like that extra line break.